dashboard/app/models/user.rb

```diff
@@ -312,6 +303,21 @@ def make_teachers_21
     self.age = 21
   end
 
+  def normalize_email
+    return unless email.present?
+    self.email = email.strip.downcase
+  end
```

Comment:
This is being done to prevent regressions to the invariant being created by the oneoff fix_emails_and_hashed_emails script.